resources/assets/uad_lists.json

```
@@ -12503,20 +12503,20 @@
   {
     "id": "com.lge.eula",
     "list": "Oem",
-    "description": "LG EULA (Terms of Use) accessible in the settings.\nEULA = https://en.wikipedia.org/wiki/End-user_license_agreement\n",
+    "description": "LG EULA (Terms of Use) accessible in the settings.\nEULA = https://en.wikipedia.org/wiki/End-user_license_agreement\nNeeded by LG HD Audio Recorder or it will force close (maybe used by some other stock app?)",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Recommended"
+    "removal": "Expert"
```

```suggestion
    "removal": "Advanced"
```
I think advanced is better, as it's a stock app but can be easily replaced by a third-party one. See https://github.com/Universal-Debloater-Alliance/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen.